### ISSUE TITLE
Incorrect stats bandwidth via cli

### ISSUE THREAD

**Issue by kallisti5** [ipfs/go-ipfs#8102]
#### Version information:
$ ipfs version --all
go-ipfs version: 0.8.0
Repo version: 11
System version: arm64/linux
Golang version: go1.15.8

#### Description:
Bandwidth usage reporting extremely low on ```ipfs stats bw```
Look at this screenshot comparison...

![Screenshot from 2021-05-01 08-22-35](https://user-images.githubusercontent.com/100110/116783858-b4133880-aa56-11eb-9ef8-6517b37c341c.png)

* ipfs stats bw reports < 100 kB/s
* ipfs webui (rest API) reports 400 kB/s consistently.

iftop on the system reports ~500-600 kB/s.   Removing overhead from the equation... the numbers from the API (and the webui) feel accurate, while the numbers from ```ipfs stats bw``` feel painfully low.

**Comment by kallisti5** [comment]
Ugh. ok.  I increased speeds and figured out what's going on.

```
$ ipfs stats bw
Bandwidth
TotalIn: 4.3 GB
TotalOut: 1.0 GB
RateIn: 5.2 MB/s
RateOut: 23 kB/s
```

* webui is reporting ```Kb/s, Mb/s```   (aka kilobits per second, Megabits per second)
* cli is reporting ```kB/s, MB/s```  (aka, kilobytes per second, Megabytes per second)

Units are important :-)

 The cli needs adjusted to KiB/s, MiB/s, GiB/s  (1024 base) *or*  KB/s, MB/s, GB/s  (1000 base)  to actually be a valid storage size.

tldr; kB isn't any standard I know of ^_^

**Comment by Stebalien** [comment]
This all seems right. `iftop` reports bits, not bytes, and `ipfs stats bw` reports the sum of data sent over streams, ignoring a lot of padding/overhead (unfortunately).
